docs/en/reference/batch-processing.rst

```diff
@@ -23,7 +23,7 @@ especially what the strategies presented here provide help with.
 .. code-block:: php
 
     <?php
-    $em->getConnection()->getConfiguration()->setSQLLogger(null);
+    $em->getConnection()->getConfiguration()->setMiddlewares([]);
```

Comment:
Maybe it could be cool to show both ways?
```suggestion
    $em->getConnection()->getConfiguration()->setMiddlewares([]); // DBAL 3
    $em->getConnection()->getConfiguration()->setSQLLogger(null); //DBAL 2
```